### ISSUE TITLE
BUG: Settings Button not working

### ISSUE THREAD

**Issue by kaduuuken** [django-wiki/django-wiki#676]
When clicking on the 'Settings' button of an article, it gives the error message:

```
AttributeError at /_settings/
type object 'Settings' has no attribute 'all'
```

Here is the Traceback:

```
Environment:


Request Method: GET
Request URL: http://127.0.0.1:8000/_settings/

Django Version: 1.11.4
Python Version: 3.6.1
Installed Applications:
('django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'django.contrib.sites',
 'django.contrib.humanize',
 'django.contrib.flatpages',
 'django_nyt',
 'mptt',
 'sekizai',
 'sorl.thumbnail',
 'wiki',
 'wiki.plugins.attachments',
 'wiki.plugins.notifications',
 'wiki.plugins.images',
 'wiki.plugins.macros',
 'allauth',
 'allauth.account',
 'allauth.socialaccount',
 'allauth.socialaccount.providers.slack',
 'slack_chat_connector',
 'activator',
 'raven.contrib.django.raven_compat',
 'anfema_groups')
Installed Middleware:
('raven.contrib.django.middleware.SentryMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'django.contrib.flatpages.middleware.FlatpageFallbackMiddleware')



Traceback:

File "/Users/k.wende/python/wiki/lib/python3.6/site-packages/django/core/handlers/exception.py" in inner
  41.             response = get_response(request)

File "/Users/k.wende/python/wiki/lib/python3.6/site-packages/django/core/handlers/base.py" in _legacy_get_response
  249.             response = self._get_response(request)

File "/Users/k.wende/python/wiki/lib/python3.6/site-packages/django/core/handlers/base.py" in _get_response
  187.                 response = self.process_exception_by_middleware(e, request)

File "/Users/k.wende/python/wiki/lib/python3.6/site-packages/django/core/handlers/base.py" in _get_response
  185.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/Users/k.wende/python/wiki/lib/python3.6/site-packages/django/views/generic/base.py" in view
  68.             return self.dispatch(request, *args, **kwargs)

File "/Users/k.wende/python/wiki/lib/python3.6/site-packages/django/utils/decorators.py" in _wrapper
  67.             return bound_func(*args, **kwargs)

File "/Users/k.wende/python/wiki/lib/python3.6/site-packages/django/contrib/auth/decorators.py" in _wrapped_view
  23.                 return view_func(request, *args, **kwargs)

File "/Users/k.wende/python/wiki/lib/python3.6/site-packages/django/utils/decorators.py" in bound_func
  63.                 return func.__get__(self, type(self))(*args2, **kwargs2)

File "/Users/k.wende/python/wiki/lib/python3.6/site-packages/django/utils/decorators.py" in _wrapper
  67.             return bound_func(*args, **kwargs)

File "/Users/k.wende/python/wiki/lib/python3.6/site-packages/wiki/decorators.py" in wrapper
  160.         return func(request, article, *args, **kwargs)

File "/Users/k.wende/python/wiki/lib/python3.6/site-packages/django/utils/decorators.py" in bound_func
  63.                 return func.__get__(self, type(self))(*args2, **kwargs2)

File "/Users/k.wende/python/wiki/lib/python3.6/site-packages/wiki/views/article.py" in dispatch
  741.             **kwargs)

File "/Users/k.wende/python/wiki/lib/python3.6/site-packages/wiki/views/mixins.py" in dispatch
  35.         return super(ArticleMixin, self).dispatch(request, *args, **kwargs)

File "/Users/k.wende/python/wiki/lib/python3.6/site-packages/django/views/generic/base.py" in dispatch
  88.         return handler(request, *args, **kwargs)

File "/Users/k.wende/python/wiki/lib/python3.6/site-packages/wiki/views/article.py" in get
  791.             self.forms.append(Form(new_article, self.request))

File "/Users/k.wende/python/wiki/lib/python3.6/site-packages/wiki/plugins/notifications/forms.py" in __init__
  157.         super(SubscriptionForm, self).__init__(*args, **kwargs)

File "/Users/k.wende/python/wiki/lib/python3.6/site-packages/django/forms/forms.py" in __init__
  96.         self.fields = copy.deepcopy(self.base_fields)

File "/Users/k.wende/python/wiki/lib/python3.6/copy.py" in deepcopy
  180.                     y = _reconstruct(x, memo, *rv)

File "/Users/k.wende/python/wiki/lib/python3.6/copy.py" in _reconstruct
  306.                 value = deepcopy(value, memo)

File "/Users/k.wende/python/wiki/lib/python3.6/copy.py" in deepcopy
  161.                 y = copier(memo)

File "/Users/k.wende/python/wiki/lib/python3.6/site-packages/django/forms/models.py" in __deepcopy__
  1186.             result.queryset = self.queryset.all()

Exception Type: AttributeError at /_settings/
Exception Value: type object 'Settings' has no attribute 'all'
```

**Comment by nschneid** [comment]
Having the same problem. I suspect it's a Django 1.11 compatibility issue.

**Comment by teury** [comment]
Please use django 10.4
For the moment

**Comment by olymk2** [comment]
just hit this as well, good to know its not just me :)

**Comment by benjaoming** [comment]
This is most likely fixed, and 0.3b3 has been released with said fix by @floemker - however, there might be another issue, see: #687
